src/system_data.jl

```diff
@@ -332,25 +332,18 @@ Call `collect` on the result to get an array.
 - `data::SystemData`: system
 - `filter_func = nothing`: Only return time_series for which this returns true.
 - `type = nothing`: Only return time_series with this type.
-- `initial_time = nothing`: Only return time_series matching this value.
 - `name = nothing`: Only return time_series matching this value.
 """
 function get_time_series_multiple(
```

Comment:
I don't know if we need to keep this function. Will just return all the `TimeSeriesData` objects or the data stored in the forecasts?

Reply:
It returns all TimeSeriesData objects